bin/i18n/i18n_script_utils.rb

```diff
@@ -93,7 +93,9 @@ def self.report_malformed_restoration(key, translation, file_name)
 
   def self.upload_malformed_restorations(locale)
     return if @malformed_restorations.blank?
-    Google::Drive.new.add_sheet_to_spreadsheet(@malformed_restorations, "i18n_bad_translations", locale)
+    if CDO.gdrive_export_secret
+      Google::Drive.new(service_account_key: StringIO.new(CDO.gdrive_export_secret.to_json)).add_sheet_to_spreadsheet(@malformed_restorations, "i18n_bad_translations", locale)
+    end
```

Comment:
I think we generally prefer one-line if statements, like 
```suggestion
    Google::Drive.new(service_account_key: StringIO.new(CDO.gdrive_export_secret.to_json)).add_sheet_to_spreadsheet(@malformed_restorations, "i18n_bad_translations", locale) if CDO.gdrive_export_secret
```

but that does result in a super long line.

What if we did something like this

```suggestion
    if CDO.gdrive_export_secret
      @google_drive ||= Google::Drive.new(service_account_key: StringIO.new(CDO.gdrive_export_secret.to_json))
      @google_drive.add_sheet_to_spreadsheet(@malformed_restorations, "i18n_bad_translations", locale)
    end
```